File: client/iOS/chalkak/chalkak/DesignSystem/Components/Logo/ChalkakLogo.swift
import SwiftUI

struct ChalkakLogo: View {
    @Environment(\.chalkakTheme) private var theme
    var body: some View {
        Text("Chalkak")
            .font(theme.typography.brand)
            .tracking(-0.42)
            .foregroundStyle(theme.colors.textPrimary)
            .accessibilityLabel("찰칵")
    }
}

#Preview("Logo") {
    ChalkakLogo()
        .padding()
}

File: client/android/app/src/main/java/com/stonefive/chalkak/core/designsystem/component/logo/ChalkakLogo.kt
package com.stonefive.chalkak.core.designsystem.component.logo

import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.tooling.preview.Preview
import com.stonefive.chalkak.core.designsystem.theme.ChalkakTheme

@Composable
fun ChalkakLogo(modifier: Modifier = Modifier) {
    Text(
        text = "Chalkak",
        modifier = modifier,
        color = ChalkakTheme.colors.textPrimary,
        style = ChalkakTheme.typography.brand,
    )
}

@Preview(showBackground = true)
@Composable
private fun ChalkakLogoPreview() {
    ChalkakTheme {
        ChalkakLogo()
    }
}
